crates/compiler/builtins/roc/Str.roc

```diff
@@ -140,40 +140,44 @@ Utf8Problem : { byteIndex : Nat, problem : Utf8ByteProblem }
 
 ## Returns `Bool.true` if the string is empty, and `Bool.false` otherwise.
 ##
-## >>> Str.isEmpty "hi!"
-##
-## >>> Str.isEmpty ""
+##     expect Str.isEmpty "hi!" == Bool.false
+##     expect Str.isEmpty "" == Bool.true
 isEmpty : Str -> Bool
+
+## Concatenate two [Str] values together. 
+##
+## >>> expect Str.concat "Hello" "World" == "HelloWorld"
```

Comment:
The use of >>> seems inconsistent.

Reply:
Thanks, I missed some. Should be all gone now.